radixdlt-java/src/main/java/com/radixdlt/fees/FeeTable.java

```diff
@@ -50,6 +55,15 @@ public UInt256 minimumFee() {
 		return this.feeEntries;
 	}
 
+	public UInt256 feeFor(Atom atom) {
+		final int feeSize = Serialize.getInstance().toDson(atom, DsonOutput.Output.HASH).length;
```

Comment:
I'm not entirely sure about moving this to FeeTable, another option I've considered is to expose this method in FeeProcessor

Reply:
Name for `feeSize` does not look good to me. At first look I've considered it somehow related to fee itself rather that part of fee related to atom size. Perhaps something like 'atomSize' would be more clear and will better reflect the value.